### PR TITLE
Refine docs about the output of bypassed nodes

### PR DESCRIPTION
**Pull request by Jasmin3q**
# Description

- Refine docs about the output of bypassed nodes.
- Add changelog.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/how-to-guides/add-conditional-control-to-a-flow.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ activate:
 ::::
 
 ### Further details and important notes
-1. If the node using the python tool has an input that references a node that may be bypassed, please provide a default value for this input. This helps prevent errors in the current node due to missing parameters.
+1. If the node using the python tool has an input that references a node that may be bypassed, please provide a default value for this input. This helps prevent errors in the current node due to missing parameters. If there is no default value for input, the output of the bypassed node will be set to None.
```
Comment (PeiwenGaoMS) [inline docs/how-to-guides/add-conditional-control-to-a-flow.md:50]: We will provide None for parameters that don't have default value, so this won't cause a missing parameter error. Please update here too.

File: src/promptflow/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,11 @@
 ### Bugs Fixed
 
 - [SDK/CLI] Keep original format in run output.jsonl.
+- [Executor] Fix failure when node refers bypassed node.
```
Comment (thy09) [inline src/promptflow/CHANGELOG.md:13]: When aggregation node refers bypassed node

File: src/promptflow/CHANGELOG.md [hunk 3]
```diff
@@ -118,4 +123,4 @@
 
 ## 0.1.0b1 (2023.07.20)
 
-- Stub version in Pypi.
```
Comment (thy09) [inline src/promptflow/CHANGELOG.md:121]: Why we changed this line?

File: src/promptflow/CHANGELOG.md (hunk 2 above)
Comment (Jasmin3q) [reply src/promptflow/CHANGELOG.md:13]: All the changes have been moved to https://github.com/microsoft/promptflow/compare/main...release/promptflow/1.0.0

File: docs/how-to-guides/add-conditional-control-to-a-flow.md (hunk 1 above)
Comment (Jasmin3q) [reply docs/how-to-guides/add-conditional-control-to-a-flow.md]: Updated.

**Review by PeiwenGaoMS — APPROVED** [review]

**Review by thy09 — APPROVED** [review]

**Pull request merged**